contribute.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "buildhub",
+    "description": "Firefox Build Metadata Service",
+    "participate": {
+        "irc": "irc://irc.mozilla.org/#storage-team",
```

Comment:
we have a `#product-delivery` channel, I think we should agree on using only one /cc @mostlygeek

Reply:
So what should I put?

Reply:
Leave it like this I guess

Reply:
left it as is :)